x-pack/plugins/reporting/common/constants/report_types.ts

```diff
@@ -5,7 +5,9 @@
  * 2.0.
  */
 
+// Export Type Definitions
 export const CSV_REPORT_TYPE = 'CSV';
+export const CSV_REPORT_TYPE_V2 = 'csv_v2';
```

Comment:
Is the new endpoint a `v2` as a progression of `v1` where `v1` should be deprecated? Or is it should be looked at as an alternative to the existing `csv`  with a different set of features? If later, then maybe `csv_v2` is not the best name for this one?